manifests/init.pp

```diff
@@ -159,13 +159,15 @@
   if $fpm {
     Anchor['php::begin'] ->
       class { '::php::fpm':
+        ensure           => $ensure,
```

Comment:
Doesn't look like it... would there be any other reason to leave them in there?

There are 5 php::fpm parameters that aren't set by ::php currently. Do you think those should be stripped out, then the remaining 7 default to undef, and pass through the values from ::php?